backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/AmendingLawService.java

```diff
@@ -73,6 +78,13 @@ public Optional<String> loadAmendingLawXml(LoadAmendingLawXmlUseCase.Query query
 
   @Override
   public Optional<String> updateAmendingLawXml(UpdateAmendingLawXmlUseCase.Query query) {
+    var updatedNorm = Norm.builder().document(XmlMapper.toDocument(query.xml())).build();
```

Comment:
Curious: do we have a (silent) agreement on `var`?

Reply:
I'm not aware that we have any agreement on using (or not using) it